plugin/hosts/hostsfile.go

```diff
@@ -46,6 +66,9 @@ type hostsMap struct {
 	byAddr map[string][]string
 }
 
+var durationOf0s, _ = time.ParseDuration("0s")
```

Comment:
nit: These can be constants instead of variables

```go
const durationOf0s = 0 * time.Second
const durationOf5s = 5 * time.Second
```

Reply:
I thought it was a good idea, so I attempted to change the code but we use the the address of the variable in the code .. I could change the struct to not use a pointer. Not sure what is best practice.

Reply:
@pmoroney happy with a pointer or should it be changed to just a time.Duration ?

Reply:
removed the pointer.